### PR TITLE
add xmlschema to rosdistro

### PR DESCRIPTION
**Pull request by hakuturu583**
I want to add xmlschema to rosdistro.

Pypy package: https://pypi.org/project/xmlschema/

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5270,6 +5270,16 @@ python-xmlplain-pip:
   ubuntu:
     pip:
       packages: [xmlplain]
+python-xmlschema:
```
Comment (tfoote) [inline rosdep/python.yaml:5273]: If this is a pip only package it should have the `-pip` suffix. 

And while we're at it these days most platforms are on python3 now so we should make this rule more future consistent using python3 in the key as pip will choose the local environment anyway.

```suggestion
python3-xmlschema-pip:
```

**Review by tfoote — CHANGES_REQUESTED** [review]
Looking at this we generally prefer to use mature natively packaged libraries so that they can be released and used in packages. Pip entries are only useful for from source builds. Is there any more formally packaged alternative?

**Comment by hidmic** [comment]
@hakuturu583 friendly ping

**Comment by dirk-thomas** [comment]
Closing to due no response from the author @hakuturu583.

**Pull request closed without merging**